### ISSUE TITLE
[BUG] RomFS mode not working

### ISSUE THREAD

**Issue by Nehemek** [TurtleP/LovePotion#125]
When running LovePotion in a fused mode created by the lovebrew tool or the 3dsxtool it will still try to locate the game directory and as it fails to do so will stop executing with an error.

System: 3ds v11.14.0-46U
CFW: Luma v10.2.1
Method of execution: Homebrew App
LÖVE Potion version: 2.0.0

**Comment by TurtleP** [comment]
ref error:
```
boot.lua: 704: Cannot load game at path 'sdmc:/3ds/game'.
Make sure a folder exists at the specified path.

Traceback

[C]: in function 'error'
[C]: in function 'xpcall'
[C]: in function 'xpcall'
```
will look into this more, as it's high priority

**Comment by TurtleP** [comment]
Seems that looking at LÖVE's logic for this, I'm following it, but since we use RomFS and not a straight-up zip file fused into the executable I have to.. change it up a little. I'll do some testing here soon when I can to fix this.

**Comment by TurtleP** [comment]
Fixed in commit 689aa40
